pkg/status/status.go

```diff
@@ -262,6 +269,40 @@ func (s *status) updateStatus(previousStatus *configv1.ClusterOperatorStatus) er
 	return err
 }
 
+// setRelatedObjects populates RelatedObjects in the ClusterOperator.Status.
+// RelatedObjects are consumed by https://github.com/openshift/must-gather
+func (s *status) setRelatedObjects() {
+	// Add the operator's namespace which will result in core resources being gathered
+	objectReferences := []configv1.ObjectReference{
+		{
+			Resource: "namespace",
+			Name:     s.namespace,
+		},
+	}
+
+	// Add the default OperatorSources and it's child resources so that their info is gathered too
+	sources := []string{"redhat-operators", "certified-operators", "community-operators"}
```

Comment:
Is there a way we can avoid using hard-coded string values here?

Reply:
Not easily. One way is to read the `manifest` folder and figure out the default `OperatorSources`. Alternatively we update the `ClusterOperator` CR every time a new marketplace CR is added. I prefer not going down either of these paths as this is a candidate for 4.1.z.